src/Test.UnitTests.Sarif/TestData/PrereleaseCompatibilityTransformer/ExpectedOutputs/ThreadFlowLocationWithKind.sarif

```diff
@@ -0,0 +1,266 @@
+          "executionSuccessful": true
+        }
+      ],
+      "artifacts": [
+        {
+          "location": {
+            "uri": "file:///e:/src/sarif/src/test.functionaltests.sarif/convertertestdata/prefast/src/relevantkeyevents/relevantkeyevents.cpp",
+            "index": 0
+          },
+          "roles": ["analysisTarget","resultFile"]
+        }
+      ],
+      "results": [
+        {
+          "ruleId": "6385",
+          "message": {
+            "text": "Reading invalid data from 'a':  the readable size is '5' bytes, but 'i' bytes may be read."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file:///e:/src/sarif/src/test.functionaltests.sarif/convertertestdata/prefast/src/relevantkeyevents/relevantkeyevents.cpp",
+                  "index": 0
+                },
+                "region": {
+                  "startLine": 8,
+                  "startColumn": 8
+                }
+              },
+              "logicalLocations": [
+                {
+                  "fullyQualifiedName": "void __cdecl C6385_Example(int)"
+                }
+              ]
+            }
+          ],
+          "codeFlows": [
+            {
+              "threadFlows": [
+                {
+                  "locations": [
+                    {
+                      "location": {
+                        "physicalLocation": {
+                          "artifactLocation": {
+                            "uri": "file:///e:/src/sarif/src/test.functionaltests.sarif/convertertestdata/prefast/src/relevantkeyevents/relevantkeyevents.cpp",
+                            "index": 0
+                          },
+                          "region": {
+                            "startLine": 3,
+                            "startColumn": 8
+                          }
+                        },
+                        "message": {
+                          "text": "'a' is an array of 5 elements (5 bytes)"
+                        }
+                      },
+                      "kinds": [
+                        "declaration"
+                      ],
```

Comment:
tadaa!

Reply:
+1

---
In reply to: [288371662](https://github.com/microsoft/sarif-sdk/pull/1510#discussion_r288371662) [](ancestors = 288371662)